helm/nginx-ingress-controller-app/templates/controller-service-internal.yaml

```diff
@@ -29,13 +29,19 @@ spec:
   {{- end }}
   ports:
   - name: http
-    port: 80
+    port: {{ .Values.controller.service.internal.ports.http }}
     protocol: TCP
-    targetPort: 80
+    targetPort: {{ .Values.controller.service.targetPorts.http }}
```

Comment:
It does not have `internal`, is it good?

Reply:
There's single shared deployment behind default and "internal" Service variant.
It's hard to express with config key names that it's target ports for the Service to use, not target ports defining the Service itself.

Reply:
ah ok got it, fine

Reply:
Also I'm trying to stay aligned with upstream when not having reason not to.

Hope that makes sense.